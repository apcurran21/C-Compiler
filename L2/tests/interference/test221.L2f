(@sqrt
	1
	%var <- rdi
	%i <- 0
	:call_label_ciao__ciao_13
	%i += 1
	%i2 <- %i
	%i2 *= %i
	%sqrt_flag <- %i2 <= %var
	cjump %sqrt_flag = 1 :call_label_ciao__ciao_13
	goto :call_label_ciao__ciao_14
	:call_label_ciao__ciao_14
	rax <- %i
	return
)
