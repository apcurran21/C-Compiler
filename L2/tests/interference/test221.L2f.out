%i2 %i r12 r13 r14 r15 rbp rbx %var
%i %i2 r12 r13 r14 r15 rbp rbx %sqrt_flag %var
r10 r11 r12 r13 r14 r15 r8 r9 rax rbp rbx rcx rdi rdx rsi
r11 r10 r12 r13 r14 r15 r8 r9 rax rbp rbx rcx rdi rdx rsi
r12 %i %i2 r10 r11 r13 r14 r15 r8 r9 rax rbp rbx rcx rdi rdx rsi %sqrt_flag %var
r13 %i %i2 r10 r11 r12 r14 r15 r8 r9 rax rbp rbx rcx rdi rdx rsi %sqrt_flag %var
r14 %i %i2 r10 r11 r12 r13 r15 r8 r9 rax rbp rbx rcx rdi rdx rsi %sqrt_flag %var
r15 %i %i2 r10 r11 r12 r13 r14 r8 r9 rax rbp rbx rcx rdi rdx rsi %sqrt_flag %var
r8 r10 r11 r12 r13 r14 r15 r9 rax rbp rbx rcx rdi rdx rsi
r9 r10 r11 r12 r13 r14 r15 r8 rax rbp rbx rcx rdi rdx rsi
rax r10 r11 r12 r13 r14 r15 r8 r9 rbp rbx rcx rdi rdx rsi
rbp %i %i2 r10 r11 r12 r13 r14 r15 r8 r9 rax rbx rcx rdi rdx rsi %sqrt_flag %var
rbx %i %i2 r10 r11 r12 r13 r14 r15 r8 r9 rax rbp rcx rdi rdx rsi %sqrt_flag %var
rcx r10 r11 r12 r13 r14 r15 r8 r9 rax rbp rbx rdi rdx rsi
rdi r10 r11 r12 r13 r14 r15 r8 r9 rax rbp rbx rcx rdx rsi
rdx r10 r11 r12 r13 r14 r15 r8 r9 rax rbp rbx rcx rdi rsi
rsi r10 r11 r12 r13 r14 r15 r8 r9 rax rbp rbx rcx rdi rdx
%sqrt_flag %i r12 r13 r14 r15 rbp rbx %var
%var %i %i2 r12 r13 r14 r15 rbp rbx %sqrt_flag
